package.json

```diff
@@ -13,9 +13,9 @@
     "test:single": "npx vitest -t"
   },
   "devDependencies": {
-    "@biomejs/biome": "^1.9.4",
     "@testing-library/dom": "^10.4.1",
     "@testing-library/jest-dom": "^6.6.4",
+    "@biomejs/biome": "^2.1.2",
```

Comment:
and there was me thinking you were well into stuff like this being alphabetical :stuck_out_tongue_winking_eye: